### ISSUE TITLE
Tie translations to gulp.src location, not location of Gulpfile

### ISSUE THREAD

**Issue by mor10** [rasmusbe/gulp-wp-pot#34]
When the gulpfile setup lives outside the project folder, the translation file gets messed up.

Let's say for example I have the following setup:

Gulpfile etc sits in
wp-content/themes/gulp-dev

Actual theme sits in 
wp-content/themes/mytheme

The Gulp task looks like this (simplified):
```javascript
var root = '../mytheme/';
var languages = root + 'languages/';

gulp.task('i18n', function () {
    return gulp.src(root + '**/*.php')
	.pipe(sort())
	.pipe(wppot( {
		domain: 'mytheme',
		destFile:  languages + 'mytheme.pot',
		package: themename
	} ))
	.pipe(gulp.dest(languages));
});
```

The resulting output in the .pot file gets this:
```
#: ../mytheme/404.php:17
msgid "hmmm. Oops! That page can&rsquo;t be found."
msgstr ""
```

The URL reference contains the relative link to the theme folder. It still works (because the URI just points back to itself) but it's not clean.

**Comment by rasmusbe** [comment]
Good thinking, I'm working on a major rewrite right now and will fix this when it's released.

**Comment by rasmusbe** [comment]
Fixed in 2.0 by adding option `relativeTo`
